presto-hive/src/test/java/com/facebook/presto/hive/TestHiveClientFileMetastore.java

```diff
@@ -152,6 +152,8 @@ public void testPartitionNotReadable()
                 splitManager.getSplits(transaction.getTransactionHandle(), session, tableLayout.getHandle(), SPLIT_SCHEDULING_CONTEXT);
                 assertNotNull(SPLIT_SCHEDULING_CONTEXT.getWarningCollector());
                 assertEquals(SPLIT_SCHEDULING_CONTEXT.getWarningCollector().getWarnings().size(), 1);
+                assertEquals(SPLIT_SCHEDULING_CONTEXT.getWarningCollector().getWarnings().get(0).getMessage(), String.format("Table '%s' has '1' out of '3' partitions unreadable:\nTesting Unreadable Partition: [ds=2020-01-03]", tableName.getTableName()));
```

Comment:
@highker , how do I get the table name?  

tableName.getTableName() returns:
test.tmp_presto_test_temptable_becbe4b8516d4eedbc8f2ec61b9cf4c7

but the expected table name is:
test.tmp_presto_test_temptable_becbe4b8516d4eedbc8f2ec61b9cf4c7

Reply:
It doesn't have to be `assertEquals`. It could be `assertTrue` followed by a substring matching or something like that